inference-engine/src/inference_engine/src/cpp_interfaces/interface/ie_iplugin_internal.cpp

```diff
@@ -219,6 +225,85 @@ void IInferencePlugin::SetExeNetworkInfo(const std::shared_ptr<IExecutableNetwor
     // Set inputs/outputs and pointer to plugin manually here
     exeNetwork->setNetworkInputs(copyInfo(constMapCast(inputs)));
     exeNetwork->setNetworkOutputs(copyInfo(constMapCast(outputs)));
+
+    ngraph::ParameterVector parameters;
+    ngraph::ResultVector results;
+    std::vector<ngraph::Output<ngraph::Node>> node_outputs;
+
+    for (auto&& input : inputs) {
```

Comment:
I don't see that you change input. I think it can be const.

Reply:
`auto&& input` propagate `const` here